src/permutat.cc

```diff
@@ -1141,6 +1175,21 @@ static Obj FuncLARGEST_MOVED_POINT_PERM(Obj self, Obj perm)
     return INTOBJ_INT(LargestMovedPointPerm(perm));
 }
 
+/****************************************************************************
+**
+*F  FuncSMALLEST_MOVED_POINT_PERM( <self>, <perm> )
+*F  Smallest point moved by perm
+**
+**  GAP-level wrapper for 'SmallestMovedPointPerm'.
```

Comment:
I don't think this adds any useful information, as the 2 line implementation makes this info clear anyway. So I'd just remove it.
```suggestion
```